validator.rb

```diff
@@ -0,0 +1,19 @@
+class Validator
+
+  def is_a_word? word 
```

Comment:
I tried that originally and it didn't work. I thought it was strange, so I decided just to move on and blow it out.